eth_utils/address.py

```diff
@@ -147,38 +143,40 @@ def is_normalized_address(value):
         return value == to_normalized_address(value)
 
 
-@coerce_args_to_bytes
-@coerce_return_to_bytes
 def to_canonical_address(address):
```

Comment:
is this function and `is_canonical_address` doing what's it's supposed to? right now the tests are only for 20-byte strings, but is that still applicable since we're doing `utf-8`? or am i missing the point of these functions